keyboards/massdrop/ctrl/keymaps/rallyrabbit/rgb_matrix_user.inc

```diff
@@ -0,0 +1,105 @@
+/*
+QMK Firmware Massdrop CTRL M-AS Keymap
+Copyright (C) 2020 matthewrobo
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// !!! DO NOT ADD #pragma once !!! //
+
+/* Step 1. */
+/* Declare custom effects using the RGB_MATRIX_EFFECT macro */
+/* (note the lack of semicolon after the macro!) */
+
+RGB_MATRIX_EFFECT(FLOWER_BLOOMING)
+RGB_MATRIX_EFFECT(RAINBOW_DIAGONAL)
+RGB_MATRIX_EFFECT(BAND_SAT_DEEP_RIVER)
+RGB_MATRIX_EFFECT(BAND_SAT_DEEP_RIVER_REVERSE)
+
+/* Step 2. */
+/* Define effects inside the `RGB_MATRIX_CUSTOM_EFFECT_IMPLS` ifdef block */
+#ifdef RGB_MATRIX_CUSTOM_EFFECT_IMPLS
+
+/*
+ * Blooming Flower was taken from HorrorTroll
+ */
+#include "flower_blooming/flower_blooming.c"
```

Comment:
C files should not be `#include`'d. Either add the code in this file or use a more appropriate way of including it.